### ISSUE TITLE
Thousands of DNS lookups to gregord.kbfs.keybase.io

### ISSUE THREAD

**Issue by libeclipse** [keybase/client#6505]
What is this domain and why are there literally thousands of requests to it while keybase isn't even being used?

**Comment by maxtaco** [comment]
cc: @mmaxim on this one

**Comment by mmaxim** [comment]
That server is used for chat, and we probably can't connect to it for some reason (are you behind a proxy of some sort?) and it is just spam retrying.

**Comment by libeclipse** [comment]
@mmaxim Nope, not behind a proxy. I can ping it fine so my system can apparently access it. I was just checking the logs on my DNS server (which isn't blocking the request) and it showed tonnes of requests to that domain.

**Comment by mmaxim** [comment]
Can you send us a Keybase log? `keybase log send`.

Thanks!

**Comment by libeclipse** [comment]
Ah, just shut down my system.

After rebooting: `2834cc2d971511c759b9fc1c`.

**Comment by libeclipse** [comment]
@mmaxim What functionality exactly relies on this? Would it be okay if I blocked the request?

**Comment by mmaxim** [comment]
The main use is for chat, but other there are other small things that also depend on it.

**Comment by zackster** [comment]
Hi!  I am hosting a Keybase bot on AWS EC2 and for security purposes, I want to configure the security group inbound rules to only permit traffic from `gregord.kbfs.keybase.io`.  Are there any similar domains I should permit, and does `gregord.kbfs.keybase.io` happen to resolve to a nice convenient CIDR block?
